charts/kafka-lag-exporter/templates/030-ConfigMap.yaml

```
@@ -16,7 +16,7 @@ data:
         {{- range $cluster := .Values.clusters }}
         {
           name = "{{ $cluster.name }}"
-          bootstrap-brokers = "{{ $cluster.bootstrapBrokers }}"
+          bootstrap-brokers = {{ $cluster.bootstrapBrokers }}
```

Would this work for your use case?
```suggestion
          bootstrap-brokers = {{ $cluster.bootstrapBrokers | quote }}
```

---

It wouldn't fix the problem. I found another way to solve it. 

Thanks.